src/app/integrations/edit-page/step-configure/data.mapper.example.host.component.ts

```diff
@@ -135,27 +141,35 @@ export class DataMapperHostComponent extends FlowPage implements OnInit, OnDestr
         atlasMapping: mappings ? JSON.stringify(mappings) : '',
       },
       onSave: () => {
-        this.initializeMapper();
+        setTimeout(() => {
+          this.initializeMapper();
+        }, 10);
       },
     });
   }
 
   initializeMapper() {
+    this.cfg.mappingService.mappingUpdated$.subscribe(() => {
+      this.detector.detectChanges();
+    });
```

Comment:
If it bothers you too, we should consider updating the lint config.. although I do agree with forcing a new line at the end of each file.